src/collection/dimensions.js

```diff
@@ -266,6 +266,20 @@ fn = elesfn = ({
         };
       }
 
+      function computePaddingValues( width, paddingObject ) {
```

Comment:
`width` could be the default if it most dev usecases would use it.  `padding-relative-to` would be a different property than `padding` and would just control what `padding` as a percent would mean.

It's not a big deal if we're different than CSS.  We already are in several places, because CSS proper doesn't always fit well with graphs.